spec/lib/karafka/params/builders/metadata_spec.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+RSpec.describe Karafka::Params::Builders::Metadata do
+  let(:routing_topic) { build(:routing_topic) }
+  let(:kafka_batch) { build(:kafka_fetched_batch) }
+
+  describe '#from_kafka_batch' do
+    subject(:result) { described_class.from_kafka_batch(kafka_batch, routing_topic) }
+
+    it { is_expected.to be_a(Karafka::Params::Metadata) }
+    it { expect(result.batch_size).to eq kafka_batch.messages.count }
```

Comment:
Can we apply brackets to all? `eq(kafka_batch.messages.count)`

Reply:
there's no rule for that now and for most of the time we don;t use them

Reply:
I will prepare a rule for `rubocop-rspec` as discussed.